mk/spksrc.service.installer.dsm6

```diff
@@ -35,10 +35,19 @@ call_func ()
     if type "${FUNC}" 2>/dev/null | grep -q 'function' 2>/dev/null; then
         install_log "Begin ${FUNC}"
         LOG=$2
+        ARG=$3
```

Comment:
That is probably a good idea for the future. I don't know if we have the use of the `call_func` well documented because there are two versions in the framework which probably need to be harmonized.